### DISCUSSION TITLE
Is there a goal for meta information?

### DISCUSSION THREAD

**Discussion by Ohrldu** [ReadySetGooo/NoNameYet#9] [General]
Is there a goal for how to solve problems, like problem solving itself? I'm also interested in things that prevent problem solving from happening. Especially human psychology information.

**Comment by ReadySetGooo** [comment]
Yes. That would be goal 17, partnerships for the goals. Could also go specifically under 17.17, Organizing, but I'll put links in 17 for now to save space.
